src/EdFi.Admin.LearningStandards.Core/Services/FromCsv/CsvLearningStandardRetriever.cs

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.Collections.Async;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using EdFi.Admin.LearningStandards.Core.Services.Interfaces.FromCsv;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace EdFi.Admin.LearningStandards.Core.Services.FromCsv
+{
+    public class CsvLearningStandardsDataRetriever : ILearningStandardsCsvDataRetriever
+    {
+        private readonly ILogger<CsvLearningStandardsDataRetriever> _logger;
+        private readonly IMetaDataRetriever _metaDataRetriever;
+        private readonly ICsvFileProcessor _csvFileProcessor;
+        private readonly IDataMappingProcess _dataMappingProcess;
+
+        public CsvLearningStandardsDataRetriever(
+            ILogger<CsvLearningStandardsDataRetriever> logger,
+            IMetaDataRetriever metaDataRetriever,
+            ICsvFileProcessor csvFileProcessor,
+            IDataMappingProcess dataMappingProcess)
+        {
+            _logger = logger;
+            _metaDataRetriever = metaDataRetriever;
+            _csvFileProcessor = csvFileProcessor;
+            _dataMappingProcess = dataMappingProcess;
+        }
+        public event EventHandler<AsyncEnumerableOperationStatus> ProcessCountEvent
+        {
+            add => _processCount += value;
+            remove => _processCount -= value;
+        }
+
+        private EventHandler<AsyncEnumerableOperationStatus> _processCount;
```

Comment:
Trivial, but check the formatting guide for the location on this. Might need to be up top.